teps/0048-task-results-without-results.md

```diff
@@ -554,9 +586,9 @@ spec:
       runAfter: [ "build-image" ]
       params:
         - name: trusted-image-name
-          value: "$(tasks.build-trusted.results.image)"
+          value: "$(tasks.build-trusted.results.image || "trusted")"
```

Comment:
NIT: I think the strings "trusted" and "untrusted" here are a bit confusing, and don't really help convey how the default value would be useful. Maybe something like "my-image:latest-trusted" and "my-image:latest-untrusted" would be more indicative of the purpose?